db.go

```diff
@@ -44,6 +51,12 @@ func NewGorm(connection string, config *gorm.Config) (*gorm.DB, error) {
 }
 
 func NewDB(connection string) (*sql.DB, error) {
+	if strings.Contains(connection, "cloudsql-instance-connection-name") {
```

Comment:
could we move this to a drivers.Parse(url) (url, DialFn) - so that it is reusabe ?

Reply:
@adityathebe ^